core_backend/migrations/versions/f269c75dbf69_add_content_table.py

```diff
@@ -0,0 +1,70 @@
+"""add content table
+
+Revision ID: f269c75dbf69
+Revises: 9eaf63e424ea
+Create Date: 2024-02-18 19:38:36.817510
+
+"""
+
+from typing import Sequence, Union
+
+import pgvector
+import sqlalchemy as sa
+from alembic import op
+from app.configs.app_config import (
+    PGVECTOR_DISTANCE,
+    PGVECTOR_EF_CONSTRUCTION,
+    PGVECTOR_M,
+    PGVECTOR_VECTOR_SIZE,
+)
+
+# revision identifiers, used by Alembic.
+revision: str = "f269c75dbf69"
+down_revision: Union[str, None] = "9eaf63e424ea"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.execute("CREATE EXTENSION IF NOT EXISTS vector;")
+    op.create_table(
+        "content",
+        sa.Column("content_id", sa.Integer(), nullable=False),
+        sa.Column(
+            "content_embedding",
+            pgvector.sqlalchemy.Vector(dim=int(PGVECTOR_VECTOR_SIZE)),
+            nullable=False,
+        ),
+        sa.Column("content_title", sa.String(length=150), nullable=False),
+        sa.Column("content_text", sa.String(length=14460), nullable=False),
```

Comment:
@suzinyou , we need to find a bigger number for the content_text. 1446 made the validatinon script fail. So I increased the number.

Reply:
Ah, and the validation script doesn't use the ORM to add data?

Reply:
Could you point me to the workflow log where this failure happened?

Reply:
No it directly adds it using the session. But it doesn't really matter as the max length would still be too low.

Reply:
Here is the [link](https://github.com/IDinsight/aaq-core/actions/runs/8064419670/job/22028262652)